packages/cli/package.json

```diff
@@ -198,6 +198,7 @@
   "prettier": {
     "singleQuote": true,
     "semi": false,
+    "arrowParens": "avoid",
```

Comment:
Haha actually... could we leave this as its default? We will most likely be using the same settings in the other repos :).

Reply:
Ah yes that was a difficult one to choose hehe. No strong opinion for or against the rule but I found the parenthesis around all the `t` in our tests not to be terribly necessary. However, if we're using the same rule in other repos, I added it back for consistency.